### ISSUE TITLE
AIP-162: linter seems to incorrectly warn about recommended pattern for revision deletion

### ISSUE THREAD

**Issue by timburks** [googleapis/api-linter#741]
The api-linter produces two warnings for the attached `library_service.proto`.

```
$ api-linter library_service.proto -I third_party/api-common-protos/
- file_path: library_service.proto
  problems:
  - message: Delete methods should include the `name` field in the URI.
    location:
      start_position:
        line_number: 26
        column_number: 3
      end_position:
        line_number: 32
        column_number: 3
    rule_id: core::0135::http-uri-name
    rule_doc_uri: https://linter.aip.dev/135/http-uri-name
  - message: Custom methods should use the HTTP POST or GET method.
    location:
      start_position:
        line_number: 26
        column_number: 3
      end_position:
        line_number: 32
        column_number: 3
    rule_id: core::0136::http-method
    rule_doc_uri: https://linter.aip.dev/136/http-method
```

These seem inconsistent with the description and example in the **DeletingRevisions** section of [AIP-162](https://google.aip.dev/162). They can be temporarily disabled by changing the included `xxx-linter` annotations to `api-linter`. 

Please clarify if this is intended behavior or if the linter is inconsistent with the AIP. Thanks!

```
// library_service.proto
syntax = "proto3";

package google.cloud.sample.library.v1alpha1;

import "google/api/annotations.proto";
import "google/api/client.proto";
import "google/api/field_behavior.proto";
import "google/api/resource.proto";
import "google/protobuf/empty.proto";

option java_package = "com.google.cloud.sample.library.v1alpha1";
option java_multiple_files = true;
option java_outer_classname = "LibraryServiceProto";
option go_package = "github.com/sample/library/rpc;rpc";

// The Library service allows teams to manage books.
service Library {
  option (google.api.default_host) = "library.googleapis.com";

  // DeleteBookRevision deletes a revision of a book.
  // (-- xxx-linter: core::0136::http-method=disabled
  //     aip.dev/not-precedent: Deleting revisions requires the 'delete' method.
  //     --)
  // (-- xxx-linter: core::0135::http-uri-name=disabled
  //     aip.dev/not-precedent: The linter is missing the 'name' in the URI. --)
  rpc DeleteBookRevision(DeleteBookRevisionRequest)
      returns (google.protobuf.Empty) {
    option (google.api.http) = {
      delete: "/v1alpha1/books/*}:deleteRevision"
    };
    option (google.api.method_signature) = "name";
  }
}

// Request message for DeleteBookRevision.
message DeleteBookRevisionRequest {
  // The name of the book revision to be deleted,
  // with a revision ID explicitly included.
  //
  // Example:
  // books/mybook.yaml@c7cfa2a8
  string name = 1 [
    (google.api.field_behavior) = REQUIRED,
    (google.api.resource_reference) = { type: "library.googleapis.com/Book" }
  ];
}

// A Book describes a book in a structured way.
message Book {
  option (google.api.resource) = {
    type: "library.googleapis.com/Book"
    pattern: "books/{book}"
  };

  // Resource name.
  string name = 1;

  // A detailed description.
  string description = 3;
}
```

**Comment by lukesneeringer** [comment]
I believe the proto is correct and the linter is wrong. Will send a fix later today.
